site/js/portfolio.js

```diff
@@ -0,0 +1,26 @@
+const avatar = document.getElementById("avatar");
+avatar.addEventListener("click", function () {
+  const currentImage = this.getAttribute("src");
```

Comment:
Attention, tu risques d'avoir des surprises avec l'utilisation de `this` sans en avoir appris le fonctionnement. Attends la POO et les explications du comportement spécific selon le contexte.
Ici, tu aurais pu faire un autre sélecteur pour l'image.